plog/plog

```diff
@@ -555,13 +547,12 @@ main () {
             logext=${log#*.} # extension, preserving .gz if it exists
             full=$FS_LOG_PATH/$ref$STATION$PLOG_FULL_SUFFIX.$logext
             if [[ $log != $full ]]; then
-                pname=$(opening_process_name $log)
-                if [[ -n $pname ]]; then
-                    fatal $log is open by $pname, aborting
+                if fuser -v $log 2>&1 | grep -q " ddout$" ; then
```

Comment:
Did you intend that `fuser` to be `flock`? I think we'll have to move some things around too as [*flock(1)*](https://linux.die.net/man/1/flock) needs to wrap a command or subshell.

Reply:
I don't know. I will put more discussion on that in the main conversation below.

I am a little concerned that using `" ddout$"` as the  _regex_ may be too pedantic. Future versions of _fuser_ may behave differently. It is good to not get distracted by other appearances of `ddout` in the output, but maybe there  should be optional space after `ddout`. The current form works for FSL8, FSL9, and FSL10, probably better to stick with what works. More radical restructuring may be needed anyway if _fuser_ changes.

Reply:
> I am a little concerned that using " ddout$" as the regex may be too pedantic

You could always implement fuser yourself by scanning the `/proc` directory!

Reply:
Yes, I would rather not. :) Maybe that would be the plan if there is a change in _fuser_